### DISCUSSION TITLE
This repo needs a new owner!

### DISCUSSION THREAD

**Discussion by hparra** [larskanis/ruby-serialport#77] [Announcements]
Friends, this repo is in desperate need of a new owner.

I thought I created a deprecation notice years ago ago, but apparently this only happened in my head. Github emails were buried until recently last year. I am no longer qualified to maintain this code as I have not used ruby or done serial port programming in well over a decade. I can't even push the gem as I have lost those credentials as well as access to the related email account.

I will help whoever in the community steps up. I'm not sure who that would be, or how we should choose that person.

My sincerest apologies to the ruby and open-source communities for my negligence.

CC: @tenderlove

**Comment by hparra** [comment]
I have contacted Ruby Gems support for further guidance. Relevant: https://blog.rubygems.org/2022/01/19/rubygems-adoptions.html

**Comment by tenderlove** [comment]
I've developed a pure Ruby alternative to this gem [here](https://github.com/tenderlove/uart).  TBH I really don't want to maintain GPL code, so I don't particularly want to work on this gem. Additionally I think the pure Ruby version I've developed is easier to maintain (and I use it everyday).

@hparra since I have access on RubyGems, I can just add a new account for you with push access. I can do the same for anyone that wants to take over as well.

**Comment by tenderlove** [comment]
Additionally, I'm happy to rename my `uart` gem to `serialport`, but I absolutely don't want to maintain backwards compatibility (which is one reason I just decided to make a new gem in the first place)

**Comment by larskanis** [comment]
I'm willing to maintain this gem since I use it in some projects. See #79 for the changes I would propose for serialport-1.4.0.

@tenderlove Can you add gem push permissions for me (lars@greiz-reinsdorf.de) since @hparra lost the rubygems credentials?

**Comment by PawelPacholek** [comment]
@larskanis I guess you still do not have gem push permissions. Have you considered making and maintaining a public fork of `serialport` with your recent changes (https://github.com/hparra/ruby-serialport/pull/79) ?

**Comment by larskanis** [comment]
@tenderlove I still would like to maintain this gem, but I still don't have the permission to do so. Would you mind to add the push permission on rubygems?

**Comment by larskanis** [comment]
I just released `serialport-1.4.0.gem` [on rubygems](https://rubygems.org/gems/serialport) fixing compat with ruby-3.4. The new repository is https://github.com/larskanis/ruby-serialport . Issues are still tracked on https://github.com/hparra/ruby-serialport/issues for now.
